siliconcompiler/tools/openroad/openroad.py

```diff
@@ -68,7 +70,14 @@ def setup(chip, mode='batch'):
     chip.set('eda', tool, 'option',  step, index, option, clobber=clobber)
     chip.set('eda', tool, 'refdir',  step, index, refdir, clobber=clobber)
     chip.set('eda', tool, 'script',  step, index, refdir + script, clobber=clobber)
-    chip.set('eda', tool, 'threads', step, index, os.cpu_count(), clobber=clobber)
+
+    # normalizing thread count based on parallelism and local
+    threads = os.cpu_count()
+    if not chip.get('remote') and step in chip.getkeys('flowgraph', flow):
```

Comment:
Not sure if the `chip.get('remote')` will have the desired effect here, since I think that will be unset on the machine where this ultimately runs.

Might need to rely on something in `chip.status` to draw a distinction here, although using that seems a bit funky to use in a tool driver.

Reply:
Good point! We need to not overload 'remote' and find a different solution to changing the operating mode.